packages/components/src/components/form-progress/form-progress.scss

```diff
@@ -19,16 +19,17 @@
         display: flex;
         flex-direction: column;
         height: 6rem;
-        justify-content: space-around;
-        width: 160px;
+        width: 134px;
```

Comment:
should we use `rem` here 🙏 and why this number looks odd 🤔

Reply:
This could have been relative, but the parent div has fixed size. So this is kept with absolute value.